static/bidder-info/iqx.yaml

```diff
@@ -1,6 +1,6 @@
 endpoint: "http://rtb.iqzone.com?pid={{.SourceId}}&host={{.Host}}&pbs=1"
 maintainer:
-  email: "it@iqzone.com"
+  email: "adops@iqzone.com"
```

Comment:
sent email for verification!

Reply:
@bretg We have sent a verification email to `adops@iqzone.com`. Please reply back to that mail with 'Received' message to complete the verification process so we can move forward with this pull request.

Reply:
Pinged the IQ Zone team to respond to the email

Reply:
verified.